packages/cli/src/commands/build.ts

```diff
@@ -477,15 +478,33 @@ export default async function main(client: Client): Promise<number> {
 function expandBuild(files: string[], build: Builder): Builder[] {
   if (!build.src) return [];
 
-  let pattern = build.src;
-  if (pattern[0] === '/') {
+  if (!build.use) {
+    throw new NowBuildError({
+      code: `invalid_build_specification`,
+      message: 'Field `use` is missing in build specification',
+      link: 'https://vercel.com/docs/configuration#project/builds',
+      action: 'View Documentation',
+    });
+  }
+
+  let src = normalize(build.src || '**');
```

Comment:
I don't know what "the production behavior" is but this will change a few things that may trigger subtle bugs (or fix them, I can't tell)

- resolve `.` and `..`
- replace slashes with their platform native slash
  - if this is `vc build` could this affect windows users? Does our system expect all slashes to be normalized to linux slashes?
- an empty string results in `.` (could hide bugs?)

As per https://nodejs.org/api/path.html#pathnormalizepath